src/sql/workbench/contrib/commandLine/electron-browser/commandLine.ts

```diff
@@ -207,6 +207,29 @@ export class CommandLineWorkbenchContribution implements IWorkbenchContribution,
 		return true;
 	}
 
+	@pathHandler({
+		path: 'openConnectionDialog'
+	})
+	public async handleOpenConnectionDialog(uri: URI): Promise<boolean> {
+		try {
+			let args = this.parseProtocolArgs(uri);
+			if (!args.server) {
+				this._notificationService.warn(localize('warnServerRequired', "Cannot connect as no server information was provided"));
+				return true;
```

Comment:
return false?

(I don't think this actually matters for anything though - right?)

Reply:
No doesn't matter for this!